### PR TITLE
Day two

### PR DESCRIPTION
**Pull request by Qairkhan**
Add other pages. used NavLink and props

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -2,23 +2,25 @@ import React from "react";
...
 import s from "./Navbar.module.css";
 
+import { NavLink } from "react-router-dom";
+
 const Navbar = () => {
   return (
     <nav className={s.nav}>
       <div className={`${s.item} ${s.active}`}>
-        <a href="#">Profile</a>
+        <NavLink to="/profile" className={s.activeLink}>Profile</NavLink>
```
Comment (Dmitry-White) [inline src/components/Navbar/Navbar.jsx]: Consider creating a folder "core" at the root level (e.g. src/core) and a "constants.js" file in that folder.
In this file you will keep the common reusable constants of your project.
As for the current line, consider creating object "ROUTES" inside the "constants.js" file and put "PROFILE" as a key and the "/profile" string as value.
Do this for EVERY such route in the whole project - find them and put them inside ROUTES object under the respective name.

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -2,23 +2,25 @@ import React from "react";
...
       </div>
       <div className={`${s.item} ${s.active}`}>
-        <a href="#">Messages</a>
+        <NavLink to="/dialogs">Messages</NavLink>
       </div>
       <div className={`${s.item} ${s.active}`}>
-        <a href="#">News</a>
+        <NavLink to="/news">News</NavLink>
```
Comment (Dmitry-White) [inline src/components/Navbar/Navbar.jsx]: Consider creating a folder "core" at the root level (e.g. src/core) and a "constants.js" file in that folder.
In this file you will keep the common reusable constants of your project.
As for the current line, consider creating object "I18Y" inside the "constants.js" file, than inside this object - another object named "EN" (short for English) and put "NEWS" as a key and the "News" string as value insider EN object.
`const I18Y = {`
`   EN: {`
`      NEWS: "News,`
`    },`
`};`
Do this for EVERY such string in the whole project - find them and put them inside I18Y object under the respective name.

**Review by Dmitry-White — CHANGES_REQUESTED** [review]

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (Qairkhan) [reply src/components/Navbar/Navbar.jsx]: Done

File: src/components/Navbar/Navbar.jsx (hunk 2 above)
Comment (Qairkhan) [reply src/components/Navbar/Navbar.jsx]: Done

File: src/components/Music/Music.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import s from "./Music.module.css";
+
+const Music = () => {
+    return (
+     <div>
+       Music
```
Comment (Dmitry-White) [inline src/components/Music/Music.jsx:8]: Consider adding the I18N to all the components.

File: src/components/core/constants.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+const ROUTES = {  
+
```
Comment (Dmitry-White) [inline src/components/core/constants.js]: Consider removing the redundant empty line.

**Review by Dmitry-White — CHANGES_REQUESTED** [review]

File: src/components/core/constants.js (hunk 4 above)
Comment (Qairkhan) [reply src/components/core/constants.js]: Done

File: src/components/Music/Music.jsx (hunk 3 above)
Comment (Qairkhan) [reply src/components/Music/Music.jsx:8]: Instead of a string in this section there will be a music player or (mb) music list

**Review by Dmitry-White — APPROVED** [review]
Good job!

**Pull request merged**